interface/therapy_groups/encounter/encounter_top.php

```diff
@@ -0,0 +1,35 @@
+<?php
+// Cloned from patient_encounter.php.
```

Comment:
Need a standard gpl header (with your copyright and author tag in it). See here for example:
http://www.open-emr.org/wiki/index.php/How_to_Document_Your_Code_Properly#Page-Level_DocBlock